docs/docs/get_started/kubernetes_tutorial.md

```diff
@@ -233,151 +274,161 @@ kubectl --namespace quick-start-backend-ns  apply -f pg-service.yml
+  service type, such as a <b>LoadBalancer</b>.
+</div>
+
+The database is now available at `$(minikube ip):30001`, which we'll call the
+`REMOTE_DB_URL`.
 
-The database has no content at this point, however you can validate that everything works by simply logging in as the admin-user. Run this command to list the roles in this DB - `psql` will be used to make a connection to the database using admin credentials:
+The database has no data yet, but we can verify it works by logging in as the
+security admin and listing the users:
 
 ```bash
-export REMOTE_DB_ADMIN_USER=quick_start_admin_user
-export REMOTE_DB_ADMIN_PASSWORD=quick_start_admin_password
+export SECURITY_ADMIN_USER=security_admin_user
+export SECURITY_ADMIN_PASSWORD=security_admin_password
 export REMOTE_DB_URL=$(minikube ip):30001
 
-docker run \
-  --rm \
-  -it \
-  -e PGPASSWORD=${REMOTE_DB_ADMIN_PASSWORD} \
-  postgres:9.6 \
-    psql \
-    -U ${REMOTE_DB_ADMIN_USER} \
-    "postgres://${REMOTE_DB_URL}/postgres" \
-    -c "\du"
+docker run --rm -it -e PGPASSWORD=${SECURITY_ADMIN_PASSWORD} postgres:9.6 \
+  psql -U ${SECURITY_ADMIN_USER} "postgres://${REMOTE_DB_URL}/postgres" -c "\du"
 ```
+
 <pre>
                                           List of roles
        Role name        |                         Attributes                    
      | Member of
 ------------------------+-------------------------------------------------------
 -----+-----------
- quick_start_admin_user | Superuser, Create role, Create DB, Replication, Bypass
+ security_admin_user    | Superuser, Create role, Create DB, Replication, Bypass
  RLS | {}
 </pre>
 
-### Setup And Configure PostgreSQL Target Service
+### Create Application Database
 
-In this section, we assume that you have a PostgreSQL backend set up and ready to go. This means:
+In this section, we assume the following:
 
-+ The PostgreSQL backend is publicly available via some URL. We'll refer to this public URL by the environment variable `REMOTE_DB_URL`
-+ Admin-level database credentials exist to allow you to create the application user. We'll refer to them as the environment variables `REMOTE_DB_ADMIN_USER` and `REMOTE_DB_ADMIN_PASSWORD`
+- You already have a PostgreSQL database exposed as a Kubernetes service.
+- It's publicly available via the URL in `REMOTE_DB_URL`
+- You have admin-level database credentials
+- The `SECURITY_ADMIN_USER` and `SECURITY_ADMIN_PASSWORD` environment variables
+  hold those credentials
 
-Note that if you are using your own database server for this tutorial and it is
-not SSL-enabled, you will want to reference the [handler documentation](/docs/reference/handlers/postgres.html) to learn how to
-modify your Secretless Broker configuration to disable SSL.
+<div class="note">
+  If you're using your own database server and it's not SSL-enabled, please see
+  the <a
+  href="https://secretless.io/docs/reference/handlers/postgres.html">handler
```

Comment:
https://docs.secretless.io/Latest/en/Content/References/handlers/postgres.htm is the updated link - I am wary about using the old one bc I expect the new site to go live any minute, but it's a little bit of a race between this PR and that one now